sources/packages/backend/libs/services/src/workflow/variables/assessment-gateway.ts

```diff
@@ -41,3 +41,12 @@ export const DISBURSEMENT_SCHEDULES = "disbursementSchedules";
  * is executed for easy application consumption.
  */
 export const WORKFLOW_DATA = "workflowData";
+
```

Comment:
Please remove the extra blank line following the same from the other in the same file.